src/kaleidoscope/plugin/Qukeys.h

```diff
@@ -25,7 +25,7 @@
 // Maximum length of the pending queue
 #define QUKEYS_QUEUE_MAX 8
 // Total number of keys on the keyboard (assuming full grid)
-#define TOTAL_KEYS ROWS * COLS
+#define TOTAL_KEYS KeyboardHardware.numKeys()
```

Comment:
It feels like in this case, we should just be replacing this macro

Reply:
Alright, I will do so.